frontend/openshift/templates/nginx-runtime/s2i/bin/run

```diff
@@ -74,5 +73,6 @@ echo "RealIpFrom = ${RealIpFrom:-172.51.0.0/16}"
 echo "API_URL = ${API_URL}"
 
 sed "s~%RealIpFrom%~${RealIpFrom:-172.51.0.0/16}~g; s~%API_URL%~${API_URL}~g;" /tmp/nginx.conf.template > /etc/nginx/nginx.conf
+sed "s~%RealIpFrom%~${RealIpFrom:-172.51.0.0/16}~g; s~%API_URL%~${API_URL}~g;" /tmp/nginx.conf.template 
```

Comment:
I'm curious, why the extra `sed` command?

Reply:
I found the logging in the openshift terminal invaluable, since the logs stick around after a failure and terminal access doesn't. Beats having to debug the dc.